src/inspector_agent.cc

```diff
@@ -380,6 +388,53 @@ void AgentImpl::WaitForDisconnect() {
   inspector_->runMessageLoopOnPause(0);
 }
 
+String16 toProtocolString(v8::Local<v8::Value> value) {
```

Comment:
Minor style issue but can you name this ToProtocolString()?  We don't generally use humpBack style.